### PR TITLE
relay requirements

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

**Review by suhasHere — APPROVED** [review]

File: draft-lcurley-moq-transport.md [hunk 1]
```diff
@@ -507,6 +507,12 @@ TODO: Refer to {{priority-congestion}}. Add details describe
 relays behavior when merging or splitting streams and interactions
 with congestion response.
 
+MoQTransport encodes the delivery information for a stream via OBJECT headers ({{message-object}}).
+
+A relay MUST treat the object payload as opaque. 
+A relay MUST NOT combine, split, or otherwise modify object payloads.
+A relay SHOULD prioritize streams ({{priority-congestion}}) based on the
+send order.
```
Comment (fluffy) [inline draft-lcurley-moq-transport.md:515]: so the last part here seems like something that should not say send order but should point at that section of the draft

Comment (suhasHere) [reply draft-lcurley-moq-transport.md:515]: yes this should send-order/object priorty ({{priority-congestion}}

**Review by afrind — COMMENTED** [review]
LGTM with Suhas feedback

File: draft-lcurley-moq-transport.md [hunk 2]
```diff
@@ -507,7 +507,14 @@ TODO: Refer to {{priority-congestion}}. Add details describe
 relays behavior when merging or splitting streams and interactions
 with congestion response.
 
-A relay MAY change the send order, in which case it SHOULD update the value on the wire for future hops.
+MoQTransport encodes the delivery information for a stream via OBJECT headers ({{message-object}}).
+
+A relay MUST treat the object payload as opaque.
```
Comment (kixelated) [inline draft-lcurley-moq-transport.md:512]: These might format strangely. Should use bullet points?

**Review by kixelated — APPROVED** [review]

**Pull request merged**